lib/stages/RawReader.hpp

```diff
@@ -34,8 +41,8 @@
  *
  * @par Buffers
  * @buffer out_buf The data read from the raw file.
- *         @buffer_format VisBuffer structured
- *         @buffer_metadata VisMetadata
+ *         @buffer_format Buffer structured
+ *         @buffer_metadata Metadata
  *
  * @conf    readahead_blocks       Int. Number of blocks to advise OS to read ahead
  *                                 of current read.
```

Comment:
@tristpinsm, is the chunk size of length 3 because the `visTranspose` stage also requires it to be of length 3 when it is used in the `visFileArchive`?

Reply:
yeah the only reason it is size three is because it is used by both stages